contracts/strategies/qv-simple/QVSimpleStrategy.sol

```diff
@@ -386,7 +372,7 @@ contract QVSimpleStrategy is BaseStrategy {
         Allocator storage allocator = allocators[_sender];
 
         if (voiceCreditsToAllocate + allocator.voiceCredits > maxVoiceCreditsPerAllocator) {
-            revert INVALID();
+            revert();
```

Comment:
why was the error removed? 
`revert INVALID()`

Reply:
looks like an accident